core-java-modules/core-java-collections-maps-5/src/test/java/com/baeldung/map/mapofvsmapofentries/MapOfEntriesVsMapOfUnitTest.java

```diff
@@ -0,0 +1,38 @@
+package com.baeldung.map.mapofvsmapofentries;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.Map;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+public class MapOfEntriesVsMapOfUnitTest {
+
+    @Test
+    public void mapOf() {
```

Comment:
it's junit5 could works without `public`
```suggestion
    void mapOf() {
```

Reply:
Good suggestion. Will change it.